pkg/vmprovider/providers/vsphere2/vmlifecycle/bootstrap.go

```diff
@@ -177,6 +198,7 @@ func getBootstrapArgs(
 				if len(r.Nameservers) == 0 {
 					r.Nameservers = nameservers
 				}
+				// v1a1 only applied the global search domains to TKG VMs.
```

Comment:
Can we use some type of pattern that indicates when something is for a specific schema version? Maybe:

```go
V1ALPHA1: Only apply global search domains to TKG VMs.
```